src/Microsoft.Azure.WebJobs.ServiceBus/ServiceBusQueueAttribute.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.ServiceBus.Messaging;
+
+namespace Microsoft.Azure.WebJobs
+{
+    /// <inherit/>
+    [AttributeUsage(AttributeTargets.Parameter)]
+    public sealed class ServiceBusQueueAttribute : ServiceBusAttribute
```

Comment:
I don't understand why we're adding a separate attribute if you've already added a boolean hint. Isn't that sufficient for us to distinguish what we need to do internally? I really think we should avoid adding a new attribute for this.